File: src/main/java/com/datastax/cdm/BaseCommand.java
package com.datastax.cdm;

import com.beust.jcommander.JCommander;
import com.beust.jcommander.Parameter;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by jhaddad on 9/14/16.
 */
public class BaseCommand {

    @Parameter(names = "--help", help = true)
    private boolean help;


}

File: src/main/kotlin/com/rustyrazorblade/cdm/BaseCommand.kt
package com.rustyrazorblade.cdm

import com.beust.jcommander.JCommander
import com.beust.jcommander.Parameter
import com.beust.jcommander.Parameters

import java.util.ArrayList

/**
 * Created by jhaddad on 9/14/16.
 */
@Parameters(commandDescription = "command")
class BaseCommand {

    @Parameter(names = arrayOf("--help"), help = true)
    private val help: Boolean = false


}
